snippet_growth_calculator/views/snippet_growth_calculator_assets.xml

```diff
@@ -0,0 +1,14 @@
+<odoo>
+    <template id="snippet_growth_calculator_assets" name="Growth Calculator Assets">
+        <xpath expr="." position="inside">
+            <link rel="stylesheet" href="/snippet_growth_calculator/static/src/css/growth_calculator.css"/>
+            <script type="text/javascript" src="/snippet_growth_calculator/static/src/js/growth_calculator.js"></script>
+            <!-- CDN Bootstrap -->
+            <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/5.3.3/css/bootstrap.min.css"/>

```

Comment:
Il faut que ces fichiers soient téléchargés localement, sinon ça brise la pérennité de ce code libre en forçant la dépendance à un lien qui peut briser.